tn.sh

```diff
@@ -2,7 +2,11 @@
 #
 # Outer wrapper for invoking test.sh with config-next inside docker-compose.
 #
+
 if type realpath >/dev/null 2>&1 ; then
   cd "$(realpath -- $(dirname -- "$0"))"
 fi
-exec docker-compose -f docker-compose.yml -f docker-compose.next.yml run boulder ./test.sh "$@"
+
+source ./tools/t-helper.sh
+
+${_compose} -f docker-compose.yml -f docker-compose.next.yml run boulder ./test.sh "$@"
```

Comment:
I'm not certain, but I believe this helps us get around namespace problems in Docker. For instance, if a `./t.sh` container hasn't quite dropped the `boulder` (default) namespace it won't cause a conflict to do a `tn.sh` run because it's using this alternate `boulder_test` namespace. I believe @jsha would know for certain though.

Reply:
I don't think it's a namespace problem, more of an oversight. The container named `boulder_tests` contains logs that `test/test-caa-log-checker.sh` can retrieve and process. I don't think the solution for that should be included in this change because this change really just lets me use `t.sh` and `tn.sh` without having to make unnecessary changes each time I create a new branch.